test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -101,6 +104,37 @@ public class HttpConnectionDispatcherTests
             }
         }
 
+        [Theory]
+        [InlineData(TransportType.WebSockets, "/ws", 404)]
+        [InlineData(TransportType.ServerSentEvents, "/sse", 404)]
+        public async Task EndPointThatOnlySupportsLongPollingRejectsOtherTransports(TransportType transportType, string path, int status)
```

Comment:
I _think_ you can OR flags in an Attribute value, but I may be wrong. It's still a compile-time constant expression...